beacon_node/beacon_chain/src/chain_config.rs

```diff
@@ -92,6 +94,7 @@ impl Default for ChainConfig {
             checkpoint_sync_url_timeout: 60,
             prepare_payload_lookahead: Duration::from_secs(4),
             optimistic_finalized_sync: true,
+            disable_backfill_rate_limiting: false,
```

Comment:
Not strong opinions on this default, maybe someone else can chime in

Reply:
I think it would be a nicer user experience if we could avoid overwhelming the CPU for most users, without them having to know about this config. For validators, backfill is less critical than being able to perform its duty, so I think it would make sense to disable this for them. Plus this will be much quicker once we have @AgeManning 's [PR](https://github.com/sigp/lighthouse/pull/3943) to limit backfill only to the WSP.

Reply:
quoting @paulhauner 
> sickos can opt-in to the no-throttle life
I'll mark this as resolved